src/internal/assert.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @hidden 指定の条件でアサートする
+ * @see https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions
+ * @param condition
+ * @param errorMessage
+ * @throw `condition` が Falthy ならば `errorMessage` を内容とする例外を投げる
+ */
+export function assert(
+  condition: unknown,
+  errorMessage?: string
```

Comment:
```suggestion
  errorMessage = '[@maboroshi-inc/type-assertions] assertion error'
```

的な感じにしておくと、コンソールをパッと見で何のエラーか分かって良いかも?(ホントに?)

Reply:
@isoden これはどうしようか悩んだんだけど非公開機能だしYAGNIかなあと思ったのでした。